.vib/common/goss/scripts/check-linked-libraries.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+read -r -a files <<< "$(find "$BITNAMI_ROOT_DIR"/"$BITNAMI_APP_NAME" "$BITNAMI_ROOT_DIR"/common -type f -executable)"
+
+for file in "${files[@]}"; do
+  if [[ ! -x "$file" ]] || [[ $(ldd "$file" | grep -c "not found") -gt 0 ]]; then
+    exit 1
+  fi
```

Comment:
The file should exist and be an executable, as we have obtained from the `find` command above. You can then simplify this to:

```suggestion
  [[ $(ldd "$file" | grep -c "not found") -gt 0 ]] && exit 1
```